content/en/docs/concepts/storage/volumes.md

```diff
@@ -106,6 +106,8 @@ spec:
       fsType: ext4
 ```
 
+If the EBS volume is partitioned, you can supply the optional field `partition: "<partition number>"` to specify which parition to mount on.
```

Comment:
nit: spelling:
partition

Reply:
```suggestion
If the EBS volume is partitioned, you can supply the optional field `partition: "<partition number>"` to specify which partition to mount on.
```

@chiehmin – if you commit the above, I think we're good to merge this! :)